base_images/isce3/docker/Dockerfile

```diff
@@ -16,14 +16,15 @@ RUN conda env create -y -f "/tmp/environment.yml" \
     && find /opt/conda/ -follow -type f -name '*.js.map' -delete \
     && /opt/conda/bin/conda clean -afy
 
-# Set the conda environment for all subsequent docker RUN commands
-# When maap-py is updated and installed via our environment.yml,
-# we won't need this line unless other envrionment specific commands are added
-SHELL ["conda", "run", "-n", "isce3", "/bin/bash", "-c"]
+RUN conda init && echo "conda activate isce3" >> ~/.bashrc
 
-RUN mkdir /maap-py \
-    && git clone --single-branch --branch v3.1.5 https://github.com/MAAP-Project/maap-py.git /maap-py/ \
-    && pip install -e /maap-py/
+# Going to extract this install into the environment.yml by this next release 
+# Install maap-py library
+SHELL ["/bin/bash", "-c"]
+RUN source activate isce3 && \
+    mkdir /maap-py && \
+    git clone --single-branch --branch v3.1.5 https://github.com/MAAP-Project/maap-py.git /maap-py/ && \
+    /opt/conda/envs/isce3/bin/pip install -e /maap-py/
 
 RUN conda list
```

Comment:
Since we've switched to `conda activate ...`, I question the output of the call, `conda list`, as I worry it isn't being executed against the `isce3` environment in this case. Instead, is it executing `conda list` against the `base` environment? If this is an issue, the `conda list` commands in this Dockerfile and our other base images will need to be updated.

Reply:
@grallewellyn ☝🏽

Reply:
@anilnatha Good point, just updated and you can see the correct conda list here: https://repo.dit.maap-project.org/root/maap-workspaces/-/pipelines/6202